pkg/authentication/scram/scram.go

```diff
@@ -271,7 +271,7 @@ func createScramCredentialsSecret(getUpdateCreator secret.GetUpdateCreator, mdbO
 func readExistingCredentials(secretGetter secret.Getter, mdbObjectKey types.NamespacedName, scramCredentialsSecretName string) (scramcredentials.ScramCreds, scramcredentials.ScramCreds, error) {
 	credentialsSecret, err := secretGetter.GetSecret(types.NamespacedName{Name: scramCredentialsSecretName, Namespace: mdbObjectKey.Namespace})
 	if err != nil {
-		return scramcredentials.ScramCreds{}, scramcredentials.ScramCreds{}, err
+		return scramcredentials.ScramCreds{}, scramcredentials.ScramCreds{}, errors.Errorf("could not get secret %s/%s: %s", scramCredentialsSecretName, mdbObjectKey.Namespace, err)
```

Comment:
[q] shouldn't we invert here `scramCredentialsSecretName` and `mdbObjectKey.Namespace` for clarity?

Reply:
ah yes! Thanks for spotting that